x11-misc/spacefm/spacefm-1.0.6.ebuild

```diff
@@ -1,9 +1,9 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
This wouldn't need an EAPI-bump anyway. If you insist, please revbump.

Reply:
No, but I decided to clean it up, and add BDEPEND